This GitHub organization keeps forks of repos developed by lab members since 2018.
It lists past and present tools from the lab and [serves as a backup](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/what-happens-to-forks-when-a-repository-is-deleted-or-changes-visibility#deleting-a-public-repository).
These forks are [automatically synced](https://github.com/apps/pull) with the parent repos and are not maintained here.
If you want to create issues or send pull requests, please do so at the parent repos.

Selected tools maintained by current lab members:
* [longcallR](https://github.com/huangnengCSU/longcallR): calling SNPs from long RNA-seq reads
* [Immuannot](https://github.com/YingZhou001/Immuannot): annotating HLA and KIR genes in phased assemblies
* [compleasm](https://github.com/huangnengCSU/compleasm): a reimplementation of BUSCO for evaluating gene completeness
* [hifieval](https://github.com/hlilab/hifieval): evaluating the accuracy of long-read error correction
* [srf](https://github.com/lh3/srf): assembler for satellite DNA
* [miniprot](https://github.com/lh3/miniprot): protein-to-genome alignment
* [hifiasm-meta](https://github.com/xfengnefx/hifiasm-meta): metagenome assembler for accurate long reads
* [hifiasm](https://github.com/chhylp123/hifiasm): de novo assembler for accurate long reads
* [minigraph](https://github.com/lh3/minigraph): pangenome construction and sequence-to-graph alignment
* [minimap2](https://github.com/lh3/minimap2): aligning long reads and contigs

Tools maintained by past lab members:
* [ntsm](https://github.com/JustinChu/ntsm): sample swap detection
* [T1K](https://github.com/mourisl/T1K): HLA and KIR genotyper
* [chromap](https://github.com/haowenz/chromap): aligning short ChIP-seq, ATAC-seq or Hi-C reads

Tools developed before 2018: [samtools](https://github.com/samtools/samtools), [BWA](https://github.com/lh3/BWA) and [more](https://github.com/lh3).
